### PR TITLE
use modern Buffer methods

### PR DESCRIPTION
**Pull request by staltz**
Similar to https://github.com/dominictarr/sodium-chloride/pull/2

This requires Node.js 5.10 or higher, but that's probably totally okay since the lowest LTS right now is Node.js 10.

### CODE REVIEW

**Comment by staltz** [comment]
Other possible reviewers: @austinfrey :)

**Review by arj03 — APPROVED** [review]
Looks good to me :)

File: index.js [hunk 1]
```diff
@@ -25,14 +25,14 @@ function encodePair (msg) {
 
   if(isString(value)) {
     flags = STRING
-    value = new Buffer(value)
+    value = Buffer.from(value)
```
Comment (mixmix) [inline index.js]: Sometimes I'm really specific and say `Buffer.from(value, 'utf8')`

This is the default but I feel more secure clearly communicating to readers and locking it down?

**Review by mixmix — APPROVED** [review]

**Comment by staltz** [comment]
Works for me, thanks @mixmix

**Comment by staltz** [comment]
Looks like 2 approvals is good to merge

**Pull request merged**